styles.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+    font-family: 'Inter Bold';
+    src: url('./assets/fonts/static/Inter-Bold.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: 'Inter Regular';
+    src: url('./assets/fonts/static/Inter-Regular.ttf') format('truetype');
+}
+
+body {
+    background-color: rgb(20, 20, 20);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+.wrapper {
+    background-color: rgb(31, 31, 31);
+    width: 384px;
```

Comment:
I'd recommend `max-width` here so that this doesn't overflow horizontally on thinner screens (mobile)